packages/netlify-cms-backend-github/src/implementation.tsx

```diff
@@ -579,6 +579,22 @@ export default class GitHub implements Implementation {
     };
   }
 
+  async listReleases() {
+    try {
+      return await this.api!.listReleases();
+    } catch (e) {
+      return [];
+    }
+  }
+
+  async publishRelease(version: string) {
+    try {
+      return await this.api!.publishRelease(version);
+    } catch (e) {
+      return;
+    }
+  }
+
```

Comment:
the rest of the code prefers the .then() syntax, you should probably respect that and write it as:

```javascript
return await this.api!.listReleases().catch(e => ([]))
```